packages/react/src/__tests__/ReactStrictMode-test.js

```diff
@@ -892,4 +892,190 @@ describe('context legacy', () => {
     // Dedupe
     ReactDOM.render(<Root />, container);
   });
+
+  describe('disableLogs', () => {
+    it('disables logs once for class double render', () => {
+      spyOnDevAndProd(console, 'log');
+
+      let count = 0;
+      class Foo extends React.Component {
+        render() {
+          count++;
+          console.log('foo ' + count);
+          return null;
+        }
+      }
+
+      const container = document.createElement('div');
+      ReactDOM.render(
+        <React.StrictMode>
+          <Foo />
+        </React.StrictMode>,
+        container,
+      );
+
+      expect(count).toBe(__DEV__ ? 2 : 1);
+      // Note: we should display the first log because otherwise
+      // there is a risk of suppressing warnings when they happen,
+      // and on the next render they'd get deduplicated and ignored.
+      expect(console.log.calls.all().map(c => c.args)).toEqual([['foo 1']]);
+    });
+
+    it('disables logs once for class double ctor', () => {
+      spyOnDevAndProd(console, 'log');
+
+      let count = 0;
+      class Foo extends React.Component {
+        constructor(props) {
+          super(props);
+          count++;
+          console.log('foo ' + count);
+        }
+        render() {
+          return null;
+        }
+      }
+
+      const container = document.createElement('div');
+      ReactDOM.render(
+        <React.StrictMode>
+          <Foo />
+        </React.StrictMode>,
+        container,
+      );
+
+      expect(count).toBe(__DEV__ ? 2 : 1);
+      // Note: we should display the first log because otherwise
+      // there is a risk of suppressing warnings when they happen,
+      // and on the next render they'd get deduplicated and ignored.
+      expect(console.log.calls.all().map(c => c.args)).toEqual([['foo 1']]);
```

Comment:
What about:

```js
expect(console.log).toBeCalledTime(n);
expect(console.log).toBeCalledWith([ /* */]);
```

Reply:
I don't understand how `toBeCalledWith` works. It only shows last cal, right? I guess in this case there _is_ only one call so sounds good. But then it's annoying to change if there are multiple.